modules/ts-capi-cli.adoc

```diff
@@ -0,0 +1,34 @@
+// Module included in the following assemblies:
+//
+// * machine_management/cluster_api_machine_management/cluster-api-troubleshooting.adoc
+
+:_mod-docs-content-type: PROCEDURE
+
+[id="ts-capi-cli_{context}"]
+= Returning the intended machines when using the CLI
+
+For clusters that use the Cluster API, `oc` commands such as `oc get machine` return results for Cluster API machines.
+Because the letter `c` precedes the letter `m` alphabetically, Cluster API machines appear in the returned results before Machine API machines do.
+You can specify the type of machines to return by using the corresponding fully qualified name.
+
+.Prerequisites
+
+* You have access to the cluster using an account with `cluster-admin` permissions.
+
+* You have installed the {oc-first}.
+
+.Procedure
+
+* To list only Machine API machines, use the fully qualified name `machines.machine.openshift.io` when running the `oc get machine` command:
++
+[source,terminal]
+----
+$ oc get machines.machine.openshift.io
```

Comment:
generalize to all relevant objects, e.g. machinesets

also, NB that failure to specify FQDN when you specify the MAPI namespace is that it will show nothing (since it prefers CAPI)